jetty-core/jetty-server/src/main/config/etc/jetty-threadpool-all-virtual.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
+
+<Configure>
+  <New id="threadPool" class="org.eclipse.jetty.util.thread.VirtualThreadPool">
+    <Set name="name" property="jetty.threadPool.namePrefix" />
+  </New>
+
+  <Call class="org.slf4j.LoggerFactory" name="getLogger">
+    <Arg>org.eclipse.jetty</Arg>
+    <Call name="info">
+      <Arg>Virtual threads enabled. Using virtual threads for all Jetty tasks.</Arg>
```

Comment:
I'd remove "Jetty" here; it is redundant and feels like "only the tasks of the Jetty implementation, so not those of the application".

Reply:
@gregw this is the `VirtualThreadPool` and "all tasks" refers to the ones submitted to this pool.
There is no need to say "Jetty", it's just the task submitted to this pool.

Reply:
You use a platform thread pool on the server, but have a proxy web application using `HttpClient` that uses `VirtualThreadPool` and now we have "all Jetty tasks" but which ones, the client's or the server's?

Perhaps let's add "Virtual thread available. Using virtual threads for all tasks submitted to VirtualThreadPool@1234."

Reply:
I still don't like that, as it is unclear what tasks are submitted to the VTP.   It is kind of saying the obvious that the VTP will use virtual threads.   We need to make the distinction between using threads for tasks calling the application vs tasks for all jetty tasks.    I think the current text does that OK.

Reply:
@gregw I don't like it because there is no definition of what a "Jetty task" is, and I don't think we want to define that as it won't be useful to users.

We just need to make a difference between: everything is run by a virtual thread, or only web application code, so specifying "Jetty" is redundant because _all_ tasks will be run by virtual threads, not only the "Jetty" tasks.

Reply:
OK, but you will get a great big "I told you so" the first time we get somebody confused that they have enabled this module, but that there are platform threads calling jetty callbacks and doing jetty things.